src/coreclr-debug/util.ts

```diff
@@ -109,7 +110,34 @@ export class CoreClrDebugUtil
             });
         });
     }
- 
+
+    public spawnChildProcess(process: string, args: string[], workingDirectory: string, onStdout?: (data: Buffer) => void, onStderr?: (data: Buffer) => void): Promise<void> {
+        const promise = new Promise<void>((resolve, reject) => {
+            const child = child_process.spawn(process, args, { cwd: workingDirectory });
+
+            if (!onStdout) {
+                onStdout = (data) => { this.log(`${data}`); };
+            }
+            child.stdout.on('data', onStdout);
+
+            if (!onStderr) {
+                onStderr = (data) => { this.log(`${data}`); };
+            }
+            child.stderr.on('data', onStderr);
+            child.on('close', (code: number) => {
+                if (code != 0) {
+                    this.log(`${process} exited with error code ${code}`);;
+                    reject(new Error(code.toString()));
+                }
+                else {
+                    resolve();
+                }
+            });
```

Comment:
>}); [](start = 12, length = 3)

You may need an 'error' handler too -- from [https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options) see 'Example of checking for failed exec:'